tests/tensor_search/test_add_documents_use_existing_tensors.py

```diff
@@ -1,584 +1,640 @@
-            # Add doc normally without use_existing_tensors
-            add_res = tensor_search.add_documents(
-                config=self.config, add_docs_params=AddDocsParams(index_name=self.index_name_1,
-                                                                  docs=doc_arg, auto_refresh=True, device="cpu"))
-
-            d1 = tensor_search.get_documents_by_ids(
-                config=self.config, index_name=self.index_name_1,
-                document_ids=[doc["_id"] for doc in doc_arg], show_vectors=True)
-
-            # Then replace doc with use_existing_tensors
-            add_res = tensor_search.add_documents(
-                config=self.config, add_docs_params=AddDocsParams(index_name=self.index_name_1,
-                                                                  docs=doc_arg, auto_refresh=True,
-                                                                  use_existing_tensors=True, device="cpu"))
-
-            d2 = tensor_search.get_documents_by_ids(
-                config=self.config, index_name=self.index_name_1,
-                document_ids=[doc["_id"] for doc in doc_arg], show_vectors=True)
+        doc_2 = {
+            "text_field_2": "content 2",
+            "_id": "1"
+        }
 
-            self.assertEqual(d1, d2)
+        for index_name in [self.structured_index, self.unstructured_index]:
+            tensor_fields = None if index_name == self.structured_index else ["text_field_1", "text_field_2"]
+            with self.subTest(f"{index_name}"):
+                tensor_search.add_documents(
+                    config=self.config,
+                    add_docs_params=AddDocsParams(
+                        index_name=index_name,
+                        docs=[doc_1],
+                        tensor_fields=tensor_fields,
+                    )
+                )
+                tensor_search.add_documents(
+                    config=self.config,
+                    add_docs_params=AddDocsParams(
+                        index_name=index_name,
+                        docs=[doc_2],
+                        tensor_fields=tensor_fields,
+                        use_existing_tensors=True
+                    )
+                )
+
+                search_res = tensor_search.search(config=self.config, index_name=index_name, text="content")
+                get_doc_res = tensor_search.get_document_by_id(config=self.config, index_name=index_name,
+                                                               document_id="1", show_vectors=True)
+
+                from pprint import pprint
+                pprint(search_res)
+                pprint(get_doc_res)
+                self.assertEqual("content 2", search_res["hits"][0]["text_field_2"])
+                self.assertEqual(1, len(get_doc_res["_tensor_facets"]))
+                self.assertEqual("content 2", get_doc_res["_tensor_facets"][0]["text_field_2"])
+
+
+    # def test_use_existing_tensors_resilience(self):
```

Comment:
Please use skip flags instead of commenting out tests. This way the diff will be much cleaner

Reply:
done